### DISCUSSION TITLE
Reset To Default Function

### DISCUSSION THREAD

**Discussion by patipati88** [xanderfrangos/twinkle-tray#151] [Ideas]
Hello, Thank you for the great app :)
I am CG artist. I'm using this app for checking dark area. 
However, In CG field, It is important always be evaluated consistently.
So my suggestion, Anything function for to revert to default function? I forget sometimes to revert default value.. X(

Add, if you can, Could you Implement anything ganma option? It will easily for checking dark area on FPS and CG. 
Sorry for my poor English. Thank you :)

**Comment by xanderfrangos** [comment]
When you say "revert to default", do you mean the brightness specifically? And if so, what do you consider "default"?

**Reply by patipati88** [reply]
Hello,
On my pc, this is the first setting I installed.
I think it is defalut value. But it looks same of monitor's brightness value. So have the opportunity to save the value as the default somewhere?

Thank you ;)
![cap](https://user-images.githubusercontent.com/22727548/112397059-aa563100-8d44-11eb-9343-84ee090aeb5e.PNG)
